### ISSUE TITLE
Change URL parsing to assume external URL's by default (without http prefix)

### ISSUE THREAD

**Issue by Discordius** [TelescopeJS/Telescope#1586]
When I create a post with an external URL of the form "LessWrong.com", it tries to redirect me to the internal url /LessWrong.com, which seems not the ideal usecase. I suggest we change the URL parsing to assume external URLs by default (i.e. by default we assume LessWrong.com means an external link to http://LessWrong.com) and require internal URLs to be prefixed by a "/", which seems like the natural choice. 

The routes.js file in nova-posts should probably be fairly straightforward to modify, but I am not super familiar with URL parsing, so I expect I would break something if I would try to make this change. There probably exist premade solutions to this problem of autocompleting incomplete URLs that I am not familiar with. But if nobody else wants to take on this change, I am happy to figure out how to do it.

**Comment by SachaG** [comment]
I think the more straightforward solution would be to prefix every URL with `http://` if it doesn't already have it. I don't think internal links are a very common use case, and if they are they probably should work differently (i.e. not require filling in a URL manually).

**Comment by Discordius** [comment]
Yep, prefixing every URL with `http://` if it doesn't already have it seems reasonable. I might get around to implementing that then. Though at the time, no promises.
